radar-upload-backend/build.gradle.kts

```diff
@@ -67,6 +67,7 @@ dependencies {
     testImplementation("org.junit.jupiter:junit-jupiter:5.4.2")
     testImplementation("org.mockito:mockito-core:2.21.0")
     testImplementation("org.hamcrest:hamcrest-all:1.3")
+    testImplementation("com.nhaarman.mockitokotlin2:mockito-kotlin:2.1.0")
```

Comment:
we already have mockito in our dependencies.Maybe it is good to move all of the tests with mockitokotlin